src/main/scala/beam/router/skim/urbansim/MasterActor.scala

```diff
@@ -214,7 +214,7 @@ class MasterActor(
     val dtInSeconds = TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis() - startedAt)
     val avgRoutePerSecond = (nSuccessRoutes + nFailedRoutes).toDouble / dtInSeconds
     val msg =
```

Comment:
This should be part of the log.info or made lazy. That way it is not calculated unless INFO is actually set.

Reply:
done